stubs/inertia/webpack.mix.js

```diff
@@ -17,5 +17,10 @@ mix.js('resources/js/app.js', 'public/js')
         require('tailwindcss'),
         require('autoprefixer'),
     ])
-    .webpackConfig(require('./webpack.config'))
-    .version();
+    .webpackConfig(require('./webpack.config'));
+
+if (mix.inProduction()) {
+    mix.version();
+} else {
+    mix.sourceMaps();
```

Comment:
Source maps should be present in production and not in development.

Reply:
Source maps may be present in production with `source-map` generating strategy instead of `eval-source-map`, but commonly used in development to debug. Why do you exclude source maps in development mode and provide mapping to source files in production?

Reply:
@slavarazum isn't it usually because the assets are minified in production but not in development - therefore source maps aren't typically necessary in development?